client/layout/index.js

```diff
@@ -12,6 +13,7 @@ import { get, isFunction } from 'lodash';
  */
 import { getHistory } from '@woocommerce/navigation';
 import { getSetting } from '@woocommerce/wc-admin-settings';
+import { PLUGINS_STORE_NAME, withPluginsHydration } from '@woocommerce/data';
```

Comment:
If we are going to be using `withPluginsHydration` at the `Layout` level, we can remove the other instances of `withPluginsHydration` which are used in the OBW, Task List, and Shipping Banner. It only needs to be used once.

Reply:
Are you sure the shipping banner can have the hydration removed? I don't think it's a child of `<Layout />` as it has it's own call to `render()`: https://github.com/woocommerce/woocommerce-admin/blob/345e39b8e84020c1b2bcfba9aa30d8c10e61b732/client/wp-admin-scripts/print-shipping-label-banner/index.js#L27-L30

Reply:
Right, good call